static/monorail-2.0.yaml

```diff
@@ -650,6 +667,130 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /hooks:
+    get:
+      description: Get the RackHD hooks configuration properties.
+      operationId: hooksGetAll
+      responses:
+        200:
+          description: Successfully retrieved the hook configuration
+          schema:
+            type: object
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+      summary: Get hooks configuration
+      tags:
+      - /api/2.0
+      x-authentication-type:
+      - jwt
+      x-privileges:
+      - Read
+      - hooksRead
+    post:
+      consumes:
+      - application/json
+      description: Create RackHD hooks configuration.
+      operationId: hooksPost
+      parameters:
+      - description: configuration hooks to be created
+        in: body
+        name: body
+        required: true
+        schema:
+          $ref: '#/definitions/hooks.2.0_HooksBase'
+      responses:
+        201:
+          description: Successfully created new hooks
+          schema:
+            type: object
+        default:
```

Comment:
There is another error code is 409, it will be returned if the input url is duplicated.